go/protocols/materialize/client.go

```diff
@@ -3,9 +3,10 @@ package materialize
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
-	"math"
+	math "math"
```

Comment:
Can't help but wonder about this. Why repeat the alias?

Reply:
No reason, my IDE did this.